src/screens/referScreen/referScreen.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { Fragment } from 'react';
+import { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import { FlatList, Text, View } from 'react-native';
+import { BasicHeader, Icon, MainButton, UserListItem } from '../../components';
+import get from 'lodash/get';
+// utils
+import { getReferralsList } from '../../providers/ecency/ecency';
+import { Referral } from '../../models';
+
+// styles
+import styles from './referScreenStyles';
+
+const ReferScreen = ({ navigation }) => {
+  const intl = useIntl();
+  const [referralsList, setReferralsList] = useState<Referral[]>([]);
+  useEffect(() => {
+    _getReferralsList();
+  }, []);
+
+  console.log('-----referralsList----- : ', referralsList);
+
+  const _getReferralsList = async () => {
+    const referralsListData = await getReferralsList('good-karma');
+    setReferralsList(referralsListData as any);
+  };
+  const _renderPointsEarned = () => {
+    return (
+      <View style={styles.pointsEarnedContainer}>
+        <Text style={styles.points}>1000</Text>
```

Comment:
points earned depend on rewarded referrals, as suggested by @feruzm we can multiply rewarded number of users with 100 to show real rewarded points. We can also show estimated value of points. 

And for unrewarded ones, how about we show "Earning Pending" based on unrewarded items